pipelines/generate_predictive.py

```diff
@@ -53,7 +53,7 @@ def generate_and_save_predictions(
         description=("Do posterior prediction from a pyrenew-hew fit.")
     )
     parser.add_argument(
-        "model-run-dir",
+        "--model_run_dir",
```

Comment:
Going forward, let's follow the convention of `snake_case` for `positional_arguments` and `kebab-case` for `--named-arguments`. So here I favor keeping the argument positional but naming it `"model_run_dir"`

Reply:
see also https://github.com/CDCgov/pyrenew-hew/pull/131#discussion_r1847044727

Reply:
> Why change from a positional argument to a named argument? And why change from `-` to `_`?

Because it was the expectation of the python scripts underlying the shell scripts in `pipelines/iteration_helpers`?